### PR TITLE
docs: add readthedocs

### PR DESCRIPTION
**Pull request by elleira**

### CODE REVIEW

File: workflow/schemas/rules.schema.yaml
Comment (maehler) [inline workflow/schemas/rules.schema.yaml:1]: I will work on completing this file.

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Welcome to Twist DNA Hematology
+
+A short introduction to our awesome pipeline Twist DNA Hematology pipeline
```
Comment (maehler) [inline docs/index.md:3]: I'll rephrase this to a proper introduction.

File: workflow/schemas/rules.schema.yaml
Comment (maehler) [reply workflow/schemas/rules.schema.yaml:1]: Ok, not really needed since the only pipeline-specific rules are related to the copying of generated reference files, and these don't really need user documentation.

**Review by alvaralmstedt — APPROVED** [review]
Vad fint ni har gjort :)
